pglookout/pgutil.py

```diff
@@ -57,12 +126,15 @@ def parse_connection_string_url(url):
         fields["dbname"] = p.path[1:]
     for k, v in parse_qs(p.query).items():
         fields[k] = v[-1]
-    return fields
+    return cast(ConnectionParameterKeywords, fields)
```

Comment:
Here you're casting, but you're not validating (at runtime) that all fields exist, right? It would be nice if we had a comment (at least) indicating this. Or, even better, having runtime validation.